receiver/kafkametricsreceiver/broker_scraper.go

```diff
@@ -51,11 +58,46 @@ func (s *brokerScraper) scrape(context.Context) (pmetric.Metrics, error) {
 		s.client = client
 	}
 
+	now := pcommon.NewTimestampFromTime(time.Now())
+	rb := s.mb.NewResourceBuilder()
+	rb.SetClusterAlias(s.config.ClusterAlias)
+
 	brokers := s.client.Brokers()
+	s.mb.RecordKafkaBrokersDataPoint((now), int64(len(brokers)))
 
-	s.mb.RecordKafkaBrokersDataPoint(pcommon.NewTimestampFromTime(time.Now()), int64(len(brokers)))
+	if s.clusterAdmin == nil {
+		admin, err := newClusterAdmin(s.config.Brokers, s.saramaConfig)
+		if err != nil {
+			s.settings.Logger.Error("Error creating kafka client with admin privileges", zap.Error(err))
+			return s.mb.Emit(metadata.WithResource(rb.Emit())), scrapeErrors.Combine()
+		}
+		s.clusterAdmin = admin
+	}
+
+	for _, broker := range brokers {
+		ID := strconv.Itoa(int(broker.ID()))
+		configEntries, err := s.clusterAdmin.DescribeConfig(sarama.ConfigResource{
+			Type:        sarama.BrokerResource,
+			Name:        ID,
+			ConfigNames: []string{"log.retention.hours"},
+		})
+		if err != nil {
+			scrapeErrors.AddPartial(1, fmt.Errorf("failed to fetch log.retention.hours config for %s: %w", broker.Addr(), err))
+			continue
+		}
+		for _, config := range configEntries {
+			if config.Name != "log.retention.hours" {
+				continue
+			}
+			val, err := strconv.Atoi(config.Value)
+			if err != nil {
+				scrapeErrors.AddPartial(1, fmt.Errorf("error processing log.retention.hours for %s", broker.Addr()))
```

Comment:
```suggestion
				scrapeErrors.AddPartial(1, fmt.Errorf("error converting `log.retention.hours` for %s: value was %s", broker.Addr(), config.Value))
```
Nit: We should make it clear that the conversion is what failed, and what value failed to be converted. Alternatively, we may want to actually include the error returned from `strconv.Atoi`